### DISCUSSION TITLE
Option to add images from originals folder to an existing album with CLI or webhooks

### DISCUSSION THREAD

**Discussion by urossimke** [photoprism/photoprism#3664] [Q&A] [answered]
Dear photoprism community,

first off, really enjoying my selfhosted instance, great job to everyone.

I have a question I have not been able to find an answer for, apologies if I missed it in the documentation or somewhere else.

My goal is to have a photoprism album automatically updated with my latest iPhone images and videos.

Currently, I have an automatic process set to get my iPhone photos and videos to a photoprism album by first automatically uploading them to the server, then with cron jobs moving them to the originals folder and finally executing a docker container command to re-index the folder (docker exec photoprism photoprism index "/path/to/album"). Then comes the manual step in the web UI of selecting the folder and connecting it with an existing album, which I'd like to automate.

So to achieve my goal, I'd want to somehow automatically add the new photos and videos to an already existing album. I have a list of them when migrating them to the originals folder, so I can potentially make a script to add one by one or in bulk.

Hence my question, is there a CLI command or some kind of webhook I can make to execute such actions?

Thank you in advance for your answers.

Best regards,
Uros

**Comment by graciousgrey** [comment, marked as answer]
Hey :)

At the moment there is no cli command for this. Is there a reason why you don't use the folder view? 

  https://docs.photoprism.app/user-guide/organize/folders/

The folder albums are automatically updated when new photos are added.

**Reply by urossimke** [reply]
Hi @graciousgrey !

Thank you for your reply, I appreciate it. I assume no web hooks as well, right?

You're right, I was completely focused on getting images to albums, since that's where things are ordered and tidy; I usually source an album from multiple folders (for example a travel album would have images form smartphones, action camera, DSLR, etc).

Your suggestion works for me, it's not ideal since I'd like to have everything in albums, but works nonetheless.

Thank you again, have a great day,
Uros

**Comment by graciousgrey** [comment]
You are welcome :) 

> I assume no web hooks as well, right?

You are right, there are no webhooks for this.

We plan to introduce [conditional albums](https://github.com/photoprism/photoprism/issues/1431) at a later time. Those might be the ideal solution for your use case once they are implemented.
